views/user/_avatar.php

```diff
@@ -20,7 +20,9 @@
 
 <?php else: ?>
 
-    <img class="user-avatar-image" alt="You currently don't have a profile picture.">
+    <object type="image/svg+xml" data="<?= Url::to('@web/image/user/default_user.svg') ?>">
```

Comment:
quite sure that this will break the upload javascript, which looks for the class to update the image source.

I can fix it if you don't know how.

Reply:
Why not use `img` tag here? Any modern browser should already support SVG as `src` of `img` tag.

Reply:
> quite sure that this will break the upload javascript, which looks for the class to update the image source.
> 
> I can fix it if you don't know how.

@cebe  `class="user-avatar-image"` is used as selector in upload file javascript code?


> Why not use img tag here? Any modern browser should already support SVG as src of img tag.

@rob006 Initially I preferred this aproach but after reading 
https://www.sitepoint.com/add-svg-to-web-page/ (see point 5)
 I changed to use `object`  tag

if `class="user-avatar-image"` is used as selector in upload file javascript code, then there is one approach to go with the @rob006 

If any better. I request @cebe  to ahead

Reply:
> @rob006 Initially I preferred this aproach but after reading
> https://www.sitepoint.com/add-svg-to-web-page/ (see point 5)

This article is from  May 30, 2012.

Reply:
SVGs are fine in `<img src` nowadays.